webapp/channels/src/components/textbox/textbox.tsx

```diff
@@ -274,11 +274,11 @@ export default class Textbox extends React.PureComponent<Props> {
             textboxClassName += ' textarea--has-labels';
         }
 
-        if (this.props.preview) {
-            return (
+        return (
+            <>
```

Comment:
Instead of having both a fragment with two `.textarea-wrapper` divs inside of it, could you actually change it so that there's one wrapper div that contains both the `SuggestionBox` and the `.textbox-preview-area` div? That'll also fix an issue with the current version where you're passing `this.wrapper` to two different elements at once

Reply:
yep, I actually thought about making that change, will make the suggested changes and get back.